infra/nncc/CMakeLists.txt

```diff
@@ -135,6 +135,15 @@ option(USE_PROTOBUF_LEGACY_IMPORT "Use legacy MODULE mode import rather than CON
 # if set ON - luci libraries are static, otherwise - shared.
 option(STATIC_LUCI "Build luci as a static libraries" OFF)
 
+# Disable PIC(Position-Independent Code) option for luci related components.
+# This option might be turned ON for MCU builds.
+#
+# Enabled PIC requires additional efforts for correct linkage, such as
+# implementation of trampoline functions and support of various address tables.
+# PIC is used for dynamic libraries, MCU builds of interpreter
+# do not benefit from it, so we prefer to disable PIC.
+option(ONE_LIBRARY_NO_PIC "Disable PIC option for libraries" OFF)
```

Comment:
(optional) there is `STATIC_LUCI` above and this one is `ONE_LIBRARY_NO_PIC`.
Description is about `luci related components.` but uses `ONE` instead of `LUCI`
Maybe use `NO_PIC_LUCI` to follow namimg convention of existing one.
Or, we can use `LUCI_LIBRARY_NO_PIC` and then change `STATIC_LUCI` to `LUCI_LIBRARY_STATIC` later.

Reply:
As I understand, `NO_PIC` is applied to libs not only _luci_ but to others so `ONE_LIBRARY_NO_PIC` is appropriate.

Reply:
> As I understand, NO_PIC is applied to libs not only luci but to others so ONE_LIBRARY_NO_PIC is appropriate.

Yes, "luci-related" components include all components used in build of luci, this includes loco, hermes and some others.

> But ONE is also for runtime OneRT, there is NNCC for compiler not including runtime.

I'll use NNCC then =)